frontend/src/routes/SearchPage/components/Modals/SearchInfoModal.tsx

```diff
@@ -54,11 +54,6 @@ export const SearchInfoModal = (props: any) => {
                 width={'50%'}
                 onClose={props.onClose}
                 isOpen={props.isOpen}
-                actions={[
-                    <AcmButton key="close" variant={ButtonVariant.primary} onClick={props.onClose}>
-                        {t('search.modal.info.action.cancel')}
```

Comment:
@dislbenn I see this is merged, but let's also cleanup the string definition from `/public/locales/en/search.json`

Reply:
Will do. I'm planning on opening up an issue for going through the string translations in each repo and cleaning them up.